common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -424,7 +424,7 @@ def test_sort_topics_update(self):
         topic = [t for t in topics if t['name'] == topic_name][0]
         self.topics_page.browse_teams_for_topic(topic_name)
         browse_teams_page = BrowseTeamsPage(self.browser, self.course_id, topic)
-        self.assertTrue(browse_teams_page.is_browser_on_page())
+        browse_teams_page.wait_for_page()
```

Comment:
note to reviewers:
There is no need for explicit assertions because wait_for_page includes a promise that is_browser_on_page will return True. It will raise a BrokenPromise if that promise if broken.
Reference: http://bok-choy.readthedocs.io/en/latest/_modules/bok_choy/page_object.html#PageObject.wait_for_page